### PR TITLE
Hierarchical clustering of PCAWG samples

### PR DESCRIPTION
**Pull request by dmaziec1**
Clustering script added. 
I added the output results, which we used for uploading to allow users to review generated heat maps and other files. Please let me know if it's ok to store it here in the repository or I should move it somewhere else and provide a link.

### CODE REVIEW

File: scripts/clustering/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+**Packages**: click, pandas, matplotlib, seaborn
+
+## CLI 
+
+The help command of the tool: 
+
+```
+python3 clustering.py  --help 
+Usage: clustering.py [OPTIONS]
```
Comment (sehilyi) [inline scripts/clustering/README.md:17]: I think you can also provide here the commands you used to generate output files. (Maybe I missed it!)

**Review by sehilyi — APPROVED** [review]
These are great -- Thanks a lot for adding this!

**Comment by dmaziec1** [comment]
Hi @sehilyi, 
I addressed your comments:
- updated the README file 
- added `run_clustering.sh` -- downloads the excel file with the signatures and runs the script
- adjusted in `clustering.py` loading the excel file as a data frame

**Comment by sehilyi** [comment]
@dmaziec1 Thanks for the update! Feel free to merge!

**Pull request merged**